flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala

```diff
@@ -83,8 +83,17 @@ trait DiscoveryServerState extends SessionManager {
     }
   }
 
+  // select a number of random peers based on a random clique id
+  def getBootstrapNeighbors(): AVector[BrokerInfo] = {
```

Comment:
It seems that this function is not used.

Reply:
nice catch!! two more commits pushed